docs/protocol-spec.md

```diff
@@ -5,8 +5,8 @@ Bindle uses HTTP/2 with TLS as a transport protocol. All bodies and responses ex
 The HTTP endpoints defined above MAY exist as a subpath on a server, or in the server's root. For example, `https://example.com/v1/_i/foo` and `https://example.com/_i/foo` are both legal paths for the specification below. However, `https://example.com/_i/v1/foo` is not (or, rather, it is a legal URI for a package named `v1/foo`).
 
 HTTP Endpoints:
-- `/_i/{bindle-name}`: The path to a bindle's invoice. Note that `{bindle-name}` can be pathy. For example, `/_i/example.com/mybindle/1.2.3` is a valid path to a bindle named `example.com/mybindle/1.2.3`.
-    - `GET`: Get a bindle by name. This returns an invoice object.
+- `/_i/{bindle-name}`: The path to a bindle. Note that `{bindle-name}` can be pathy. For example, `/_i/example.com/mybindle/1.2.3` is a valid path to a bindle named `example.com/mybindle/1.2.3`.
+    - `GET`: Get a bindle by name.
```

Comment:
"Get a bindle object by name" <- Important to connect it with the new section below